src/Templates/src/templates/maui-lib/Class1.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace MauiLib1
+{
+	// All the code in this file is shared between all the platforms
+	// including net6.0 shared library.
```

Comment:
The phrasing here doesn't ring well to me. I see that you're trying to say it's both for all platforms and also in some sense "no" platform. Maybe the term `shared library` doesn't fit well? I'm not sure that's a general term?

Reply:
I updated the description to say this is always included.